homeassistant/components/kaiterra/api_data.py

```diff
@@ -55,7 +55,7 @@ async def async_update(self) -> None:
         try:
             async with async_timeout.timeout(10):
                 data = await self._api.get_latest_sensor_readings(self._devices)
-        except (ClientResponseError, asyncio.TimeoutError):
+        except (ClientResponseError, ClientConnectorError, asyncio.TimeoutError):
```

Comment:
You really should log some info on what exception occured in debug level.

Reply:
added in subsequent commit